services/gitdiff/csv.go

```diff
@@ -199,25 +181,82 @@ func createCsvDiff(diffFile *DiffFile, baseReader *csv.Reader, headReader *csv.R
 		}
 
 		for i := 0; i < len(a2b); i++ {
```

Comment:
This could do with some comments to explain what it's doing and why we're iterating over both sides.

Reply:
Yeah, it does need some commenting....especially when I shift the cells array with copy(). Actually wish the first person to write this commented...I really had to think with this one.

Reply:
yeah